app/bettermail.go

```diff
@@ -127,6 +135,72 @@ func handlePushPayload(payload PushPayload, c appengine.Context) (*mail.Message,
 	return message, nil
 }
 
+func fetchCommit(payload CommitCommentPayload, commit *ApiCommit, c appengine.Context) error {
+	commitSHA := *payload.Comment.CommitID
+	url := strings.Replace(*payload.Repo.CommitsURL, "{/sha}", "/" + commitSHA, 1)
+
+	client := urlfetch.Client(c)
```

Comment:
This won't work for private repos. I would rather not have to make the app more complex by requiring it to store auth tokens that allow read access to repos.

As an alternative design, since you should always get a `push` event before a `commit-comment` event, you could store a simple map of commit ID -> email subject. Then when a comment arrives, you would know what subject to generate. This would also let you avoid splitting multi-commit pushes into separate emails, since you would know what subject to generate for them (generating one email per commit would be annoying for merge pushes that include lots of commits).